<launch>
  <node name="dynamixel_workbench_velocity_controllers" pkg="dynamixel_workbench_controllers" type="dynamixel_workbench_velocity_control" required="true" output="screen">
    <rosparam>
      device_name: "/dev/ttyUSB0"
      baud_rate: 57600
      motor_model: "XM430_W350_R"
      protocol_version: 2.0
      left_motor:
        motor_id: 1
      right_motor:
        motor_id: 2
    </rosparam>
  </node>
</launch>
